### ISSUE TITLE
Check integration result

### ISSUE THREAD

**Issue by MLopezJ** [MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#8]
# TL;DR

Integration is working fine and flow is the one that is expected. 

There is a different in the result of device twin when data is defined as "Telemetry" and when its defined as "Property'

The integration between Thingy:91 and Coiote Dashboard is not working always as expected. Reason about this was not found.

# Issue discussion

As result from[ integration between Thingy and Azure through Coiote](https://iotdevzone.avsystem.com/docs/Demo_Projects/Tracking_tutorial/) is taken [this](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#expected-input) obejct 

But recently [it was discovered](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/7#issuecomment-1424440325) that the data shown in Coiote is not the same than the received in Azure. 

It is needed to double check Azure integration with the intention of make sure data to be the same on both sites. 

And also to have more information about the [new data struct on input data ](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/7)

Integration discussion available [here](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/1)

**Comment by MLopezJ** [comment]
# Check timestamps between results 

Compare both timestamps to know if outputs refers to the same object 

| Object | Prop | Coiote's value | Coiote's last updated | Azure twin"s value | Azure twin's last updated |
| --- | --- | --- | --- | --- | --- |
| 6 | 0 | not provided | **-** | { "pmin": "600", "pmax": "3600" } | **2023-02-08T16:45:54.4884359Z** |
| 6 | 1 | not provided | **-** | { "pmin": "600", "pmax": "3600" } | **2023-02-08T16:45:54.4884359Z** |
| 3303 | 5700 | 28.4 | **2023-02-08, 18:10:02** | { "pmin": "600", "pmax": "3600" } | **2023-02-08T16:45:54.4884359Z** |
| 3304 | 5700 | 12.713 | **2023-02-08, 17:45:53** | { "pmin": "600", "pmax": "3600" } | **2023-02-08T16:45:54.4884359Z** |
| 3315 | 5700 | 12.713 | **2023-02-08, 17:45:53** | { "pmin": "600", "pmax": "3600" } | **2023-02-08T16:45:54.4884359Z** |


As shown in **Coiote's last updated** and **Azure twin's last updated**, values are different. There can be concluded that both outputs do not refers to the same object.


# ~How to write in Azure twin ?~

# ~How to write newest Coiote's object in Azure twin ?~

**Comment by MLopezJ** [comment]
> # How to write in Azure twin ?
> # How to write newest Coiote's object in Azure twin ?

`Azure device twin` is a document with a json object. It contains its own format. Its main sections are: `desired` and `reported`

``` json
{
    "deviceId": "device1",
     ...
     "properties": {
         "desired": {
        ...
        },
        "reported": {
            ...
        }
     }  
}
```

It is created automatically when a device is added to IoT hub, so questions must be: 

# ~How to write in a IoT hub device?~

**Comment by MLopezJ** [comment]
# Check Data flow

Data flow should be: Thingy:91 --> Coiote Dashboard --> IoT Hub Device -- > Device twin 

- [x] Thingy:91 --> Coiote Dashboard
- [x] Coiote Dashboard --> IoT Hub Device
- [x] IoT Hub Device -- > Device twin

**Comment by MLopezJ** [comment]
> Thingy:91 --> Coiote Dashboard --> IoT Hub Device -- > Device twin

~**This flow is not the one that is currently operating**~ The last affirmation is not true. More info [Here](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/8#issuecomment-1428184789)

Because `Thingy:91 --> Coiote Dashboard` is working as expected but `Coiote Dashboard --> IoT Hub Device`
 is not. 

On the `IoT Hub Device` dashboard, there is not information about `last status update`, neither about `connection state` or `last activity time`. 

![Screenshot from 2023-02-10 15-05-14](https://user-images.githubusercontent.com/8351865/218111431-08e23f63-50ff-4433-a513-f9847fea1e9a.png)


![Screenshot from 2023-02-10 15-04-25](https://user-images.githubusercontent.com/8351865/218111403-e5f04447-d16b-4d9a-9613-78d79f9823e7.png)


But somehow, the device twin is updated with new data except for the props mentioned at the beginning of this document 

![Screenshot from 2023-02-10 15-03-45](https://user-images.githubusercontent.com/8351865/218111066-e03fadc3-063c-4006-90c4-61beaca2d712.png)

**Comment by MLopezJ** [comment]
# Is the `device twin` been modified by any `Power BI` function instead of by the `IoT hub device`?

No.

In the [AV System's tutorial](https://iotdevzone.avsystem.com/docs/Demo_Projects/Tracking_tutorial/), the architecture looks like the first connection in Azure is with the `IoT hub` service, and then with `Power BI`. So I'm assuming than `Power BI` can not affect `IoT Hub` service just because the order of the interactions.  

![Screenshot from 2023-02-10 15-55-24](https://user-images.githubusercontent.com/8351865/218122561-b02598c3-fe89-4f71-821a-29d6e85d3782.png)

**Comment by MLopezJ** [comment]
### Create new device on environments and test flow

- [x] Create new device in Coiote Dashboard 
  - Creattion date: 2023-02-10, 16:13:56
- [ ] Receive data in Coiote Dashboard from Thinghy:91
  - Use already generated firmware
  - Update Thingy config using anjay
  - Result: Not able to connect to Coiote after some tries
  - Re write firmware with Programmer
  - Turn off and on the Thingy
  - Update Thingy config using anjay
  - After try again result was the same
  - Update Thingy config with already connected device info
  - data are send
- [ ] Create new Azure IoT Hub
- [ ] Send data from Coiote Dashboard to new Azure IoT Hub

**Comment by MLopezJ** [comment]
> How to write in a IoT hub device?

Through the Coiote integration it is possible to write to the IoT hub device. Only needs to make the integration, turn on the Thingy 91 and data should be send to Coiote, then it will be send to the connected IoT device and replicated in the Device Twin. 

It works for the rest of objects excepts for the mentioned 5. 

Issue is not related to do not be able to write data in the IoT hub device.

**Comment by MLopezJ** [comment]
> > Thingy:91 --> Coiote Dashboard --> IoT Hub Device -- > Device twin
> 
> **This flow is not the one that is currently operating**

The last affirmation is not true. 

Flow described there is operating and working properly. I can affirm that because the rest of the objects are being update as expected. Prove of it is that in the `device twin` all the objects has in the `$lastUpdated` a value than make sense with the value showed in the Coiote Dashboard. 

> On the `IoT Hub Device` dashboard, there is not information about `last status update`, neither about `connection state` or `last activity time`.

Also, neither of those attributes are related to the device twin update status. And the counter in the device twin object is increasing on each iteration

**Comment by MLopezJ** [comment]
> Check Data flow

Flow is working as expected. 

> 
> Are those values telemetry data?
> Yes! Were set [here as Telemetry resources](https://iotdevzone.avsystem.com/docs/Demo_Projects/Tracking_tutorial/#:~:text=Set%20the%20CAPABILITY%20TYPE%20to%20Telemetry%20for%20the%20resources%3A)
> 
> What does all the values has in similar?
> Values were set as "Value Tracking" in Coiote: [link here](https://iotdevzone.avsystem.com/docs/Demo_Projects/Tracking_tutorial/#:~:text=%3E%20Devices.-,Set%20group%20value%20tracking%20on%20resources%20in%20Coiote%20DM,-%23)
> 
> The `min-max` value is present in YouTube video tutorial as well?
> Not visible on the following videos:
> - https://www.youtube.com/watch?v=dn5JAlIokA4
> - https://www.youtube.com/watch?v=FvW2WQDMaSc
> 
> _Originally posted by @MLopezJ in https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/11#issuecomment-1427967337_


By having answered the past questions a new approach to understand this issue is proposed:

- Create new integration without setting the mentioned objects as "Value Tracking" in Coiote

If that doesnt work it is suggested to:

- Create new integration without setting the mentioned objects as "Telemetry" values and "Value Tracking" in Coiote

# Solution

Problem was solved after remove device from IoT hub and remove device from Coiote Dashboard as well. Then create new device in Coiote Dashboard and **make the integration without setting the mentioned objects as "Telemetry" values and "Value Tracking" in the Coiote configuration**

It means that `{ "pmin": "600", "pmax": "3600" }` value was generated because of the way Coiote treats `Telemetry` data. It is important to keep digging on it a little bit more to complete undertand the context. 

Also, I'm assuming that `pmin` stands from `period minimum` and `pmax` from `period maximum` and that value is in seconds format. But this is just an assumption.

**Comment by MLopezJ** [comment]
# Device twin full object

``` json
{
    "deviceId": "test-a",
    "etag": "AAAAAAAAAAE=",
    "deviceEtag": "OTI0MTcyNTM5",
    "status": "enabled",
    "statusUpdateTime": "0001-01-01T00:00:00Z",
    "connectionState": "Connected",
    "lastActivityTime": "0001-01-01T00:00:00Z",
    "cloudToDeviceMessageCount": 0,
    "authenticationType": "sas",
    "x509Thumbprint": {
        "primaryThumbprint": null,
        "secondaryThumbprint": null
    },
    "modelId": "",
    "version": 4,
    "properties": {
        "desired": {
            "$metadata": {
                "$lastUpdated": "2023-02-13T17:10:20.6589393Z"
            },
            "$version": 1
        },
        "reported": {
            "lwm2m": {
                "1": {
                    "0": {
                        "0": {
                            "value": 1
                        },
                        "1": {
                            "value": 50
                        },
                        "6": {
                            "value": false
                        },
                        "7": {
                            "value": "U"
                        },
                        "16": {
                            "value": true
                        },
                        "23": {
                            "value": false
                        }
                    }
                },
                "3": {
                    "0": {
                        "0": {
                            "value": "Nordic Semiconductor"
                        },
                        "1": {
                            "value": "Thingy:91"
                        },
                        "2": {
                            "value": "351358815340515"
                        },
                        "3": {
                            "value": "22.8.1+0"
                        },
                        "11": {
                            "0": {
                                "value": 0
                            },
                            "attributes": {
                                "dim": "1"
                            }
                        },
                        "13": {
                            "value": 1676308507000
                        },
                        "16": {
                            "value": "UQ"
                        },
                        "19": {
                            "value": "3.2.1"
                        }
                    }
                },
                "4": {
                    "0": {
                        "0": {
                            "value": 6
                        },
                        "1": {
                            "0": {
                                "value": 6
                            },
                            "1": {
                                "value": 7
                            },
                            "attributes": {
                                "dim": "2"
                            }
                        },
                        "2": {
                            "value": -89
                        },
                        "3": {
                            "value": 21
                        },
                        "4": {
                            "0": {
                                "value": "10.160.54.210"
                            },
                            "attributes": {
                                "dim": "1"
                            }
                        },
                        "8": {
                            "value": 33703712
                        },
                        "9": {
                            "value": 2
                        },
                        "10": {
                            "value": 242
                        }
                    },
                    "attributes": {
                        "ver": "1.2"
                    }
                },
                "5": {
                    "0": {
                        "0": {},
                        "1": {
                            "value": ""
                        },
                        "3": {
                            "value": 0
                        },
                        "5": {
                            "value": 0
                        },
                        "8": {
                            "0": {
                                "value": 0
                            },
                            "1": {
                                "value": 1
                            },
                            "attributes": {
                                "dim": "2"
                            }
                        },
                        "9": {
                            "value": 2
                        }
                    }
                },
                "6": {
                    "0": {
                        "0": {},
                        "1": {},
                        "2": {},
                        "3": {},
                        "5": {},
                        "6": {}
                    }
                },
                "3303": {
                    "0": {
                        "5601": {
                            "value": 27.18
                        },
                        "5602": {
                            "value": 27.71
                        },
                        "5700": {
                            "value": 27.18
                        },
                        "5701": {
                            "value": "Cel"
                        }
                    }
                },
                "3304": {
                    "0": {
                        "5601": {
                            "value": 23.535
                        },
                        "5602": {
                            "value": 24.161
                        },
                        "5700": {
                            "value": 24.057
                        },
                        "5701": {
                            "value": "%RH"
                        }
                    }
                },
                "3313": {
                    "0": {
                        "5701": {
                            "value": "m/s2"
                        },
                        "5702": {
                            "value": -0.372652
                        },
                        "5703": {
                            "value": -0.117679
                        },
                        "5704": {
                            "value": -9.012311
                        }
                    }
                },
                "3315": {
                    "0": {
                        "5601": {
                            "value": 101697
                        },
                        "5602": {
                            "value": 101705
                        },
                        "5700": {
                            "value": 101705
                        },
                        "5701": {
                            "value": "Pa"
                        }
                    }
                },
                "3347": {
                    "0": {
                        "5500": {
                            "value": false
                        },
                        "5501": {
                            "value": 0
                        },
                        "5750": {
                            "value": "Button 0"
                        }
                    }
                },
                "3420": {
                    "0": {
                        "1": {
                            "value": "#000000"
                        }
                    }
                },
                "10256": {},
                "50001": {
                    "0": {
                        "0": {
                            "value": 5
                        },
                        "1": {
                            "value": 495
                        },
                        "6": {},
                        "7": {
                            "value": "403"
                        },
                        "8": {},
                        "9": {},
                        "10": {},
                        "11": {}
                    }
                }
            },
            "$metadata": {
                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                "lwm2m": {
                    "1": {
                        "0": {
                            "0": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "1": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "6": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "7": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "16": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "23": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3": {
                        "0": {
                            "0": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "1": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "2": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "3": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "11": {
                                "0": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "value": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                },
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "attributes": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "dim": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                }
                            },
                            "13": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "16": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "19": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "4": {
                        "0": {
                            "0": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "1": {
                                "0": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "value": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                },
                                "1": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "value": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                },
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "attributes": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "dim": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                }
                            },
                            "2": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "3": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "4": {
                                "0": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "value": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                },
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "attributes": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "dim": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                }
                            },
                            "8": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "9": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "10": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                        "attributes": {
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                            "ver": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            }
                        }
                    },
                    "5": {
                        "0": {
                            "0": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "1": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "3": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "8": {
                                "0": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "value": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                },
                                "1": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "value": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                },
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "attributes": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                    "dim": {
                                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                    }
                                }
                            },
                            "9": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "6": {
                        "0": {
                            "0": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "1": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "2": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "3": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "5": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "6": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3303": {
                        "0": {
                            "5601": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5602": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5700": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5701": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3304": {
                        "0": {
                            "5601": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5602": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5700": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5701": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3313": {
                        "0": {
                            "5701": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5702": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5703": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5704": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3315": {
                        "0": {
                            "5601": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5602": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5700": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5701": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3347": {
                        "0": {
                            "5500": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5501": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "5750": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "3420": {
                        "0": {
                            "1": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                                }
                            },
                            "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "10256": {
                        "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                    },
                    "50001": {
                        "0": {
                            "0": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "1": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "6": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "7": {
                                "$lastUpdated": "2023-02-13T17:15:27.2130787Z",
                                "value": {
                                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                                }
                            },
                            "8": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "9": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "10": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "11": {
                                "$lastUpdated": "2023-02-13T17:15:10.8531098Z"
                            },
                            "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                        },
                        "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                    },
                    "$lastUpdated": "2023-02-13T17:15:27.2130787Z"
                }
            },
            "$version": 3
        }
    },
    "capabilities": {
        "iotEdge": false
    }
}
```

**Comment by MLopezJ** [comment]
> Create new device on environments and test flow

It is interesting how the connection between Thingy:91 and Coiote sometimes is not working as expected. In the attempt that was made to create a new environment, this step fails and was needed to use the previous device connection

> Receive data in Coiote Dashboard from Thinghy:91
> * Use already generated firmware
> * Update Thingy config using anjay
> * Result: Not able to connect to Coiote after some tries
> * Re write firmware with Programmer
> * Turn off and on the Thingy
> * Update Thingy config using anjay
> * After try again result was the same
> * Update Thingy config with already connected device info

It is not the main objective of this project so no effort is going to be made in this concern.
